src/test/regress/input/multi_copy.source

```diff
@@ -254,3 +254,18 @@ SET citus.shard_max_size TO '256kB';
 COPY lineitem_copy_append FROM '@abs_srcdir@/data/lineitem.1.data' with delimiter '|';
 
 SELECT count(*) FROM pg_dist_shard WHERE logicalrelid = 'lineitem_copy_append'::regclass;
+
+-- Connect to the first worker node
+\c - - - 57637
+
+-- Test copy from the worker node
+COPY lineitem_copy_append FROM '@abs_srcdir@/data/lineitem.2.data' with (delimiter '|', master_host 'localhost', master_port 57636);
```

Comment:
Maybe put an index and a constraint on the table to increase test coverage and run the copy twice.

minor: maybe split these copy into 2 lines.